app/views/home/PageHome.vue

```diff
@@ -250,6 +250,15 @@ export default Vue.extend({
     BaseCloudflareVideo
   },
   data () {
+    let teacherSignupModal = true
+    let teacherModalPath = 'teacher'
+    let teacherLink = null
+    if (me.isTeacher()) {
```

Comment:
this logic should be in the created or mounted hook. Normally, it's not an issue with `me` in our code since we set window.me on initialize but we should generally set such things after component is initialized.

You can create non-reactive variables like here: https://dev.to/scottstern06/multiple-ways-to-set-non-reactive-values-on-a-vue-component-5dk9 or just set this.engagingBoxes[n].link in mounted/created

Reply:
Updated to use this.engagingBoxes[n].link